app/routers/calendar.py

```diff
@@ -1,12 +1,11 @@
 from http import HTTPStatus
 
-from fastapi import APIRouter, Request
+from app.dependencies import get_db, templates
+from app.routers import calendar_grid
```

Comment:
Separate to another paragraph, after 3rd party modules

Reply:
ok